wallet/wallet-cli-commands/src/command_handler/mod.rs

```diff
@@ -1808,6 +1820,19 @@ fn id_to_hex_string(id: H256) -> String {
     hex_string.strip_prefix("0x").unwrap_or(&hex_string).to_string()
 }
 
+fn disabled_qr_code_env() -> bool {
+    std::env::var("ML_DISABLE_QR").is_ok()
+}
+
+fn optional_qrcode_or_error_string(str_data: &str, no_qr: bool) -> String {
+    if no_qr {
+        String::new()
+    } else {
+        let qr_code_string = qrcode_or_error_string(str_data);
+        format!("\n\nOr scan the Qr code with it:\n\n{qr_code_string}")
```

Comment:
This code is very context dependant and it's quite easy to break formatting here when used incorrectly from outside. I'd duplicate some code but do this in place.

It's possible to write it in a more compact way with `no_qr.then().unwrap_or_else()`